README.md

```diff
@@ -315,6 +316,13 @@ Head over to [DataStax Academy](https://academy.datastax.com/quick-downloads?utm
 * Step-by-step tutorials and examples
 * How to build applications using Apache Cassandra™/ DataStax
 
+# Docker Configuration Tips and Tricks
+
+* If your DSE instance is stopping after starting studio or opsCenter, 
```

Comment:
IMHO, it is not clear what is the usage scenario here - what is the platform you are referring to? what OS? what do you refer as `docker memory`, and how to increase it ?

Reply:
In this example, I had a problem with MAC OS docker (docker-machine) but every OS could have a different way of changing that. `docker memory` is just a RAM assigned to docker on the host. I thought that there could be a generic section about tips when running our docker-images

Reply:
@tomekl007 If I am reading correctly, you suggesting we add a section on Tweaks and Configuring Docker itself?  From the example you provided, it appears you needed to increase the ram for Docker on Mac allocated to the Virtual Machine the containers run inside, this would also apply to Docker on Windows but not Linux.  

I get you point because I have seen other vendors reference increasing the ram allocated to docker on mac and windows.  Im not sure how deep of a rabbit hole we would want to go down with docker specific settings.

Reply:
Yes, I added that section: Docker Configuration Tips and Tricks
I think that it may reduce a lot of time for future users because I spend some time figuring out that this is a memory problem. There was no explicit information from docker, it just killed one container

Reply:
I agree with @roberd13 here - @tomekl007 you should expand this paragraph providing the details about the environment where the issue occurred. `docker-machine` is just one way of installing Docker on OSX - [current Docker for OSX](https://docs.docker.com/docker-for-mac/install/) uses HyperKit and a slightly different approach.

Reply:
Ok :) can you suggest a change? 
There is an info that `Default on OSX is 2GB - that is not enough. `

Reply:
well, just describe as precise as possible your case - I would avoid actually the very general title "Docker Configuration Tips and Tricks" - Instead perhaps just a title stating "OSX Caveats" ?

Reply:
title changed :)